deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+git fetch && git pull
+
+cd ./client
+npm ci
+npm run build:prod
```

Comment:
`client/package.json` 변경되었습니다!
```suggestion
npm run build
```

Reply:
https://github.com/boostcampwm-2022/web19-Boostart/pull/217/commits/ae8608619319b88af167a2bd361a299302007a63 수정했습니다~
감사합니다!